camera_calibration/src/camera_calibration/camera_calibrator.py

```diff
@@ -93,28 +93,24 @@ def __init__(self, name, boards, service_check = True, synchronizer = message_fi
                  max_chessboard_speed = -1, queue_size = 1):
         super().__init__(name)
 
-        left_camera = self.declare_parameter("left_camera", "left_camera").get_parameter_value().string_value
-        right_camera = self.declare_parameter("right_camera", "right_camera").get_parameter_value().string_value
-        camera = self.declare_parameter("camera", "camera").get_parameter_value().string_value
-
-        self.set_camera_info_service = self.create_client(sensor_msgs.srv.SetCameraInfo, camera + "/set_camera_info")
-        self.set_left_camera_info_service = self.create_client(sensor_msgs.srv.SetCameraInfo, left_camera + "/set_camera_info")
-        self.set_right_camera_info_service = self.create_client(sensor_msgs.srv.SetCameraInfo, right_camera + "/set_camera_info")
+        self.set_camera_info_service = self.create_client(sensor_msgs.srv.SetCameraInfo, "/camera/set_camera_info")
```

Comment:
Could you please remove the leading `/` here? This causes issues with namespaces.

Reply:
Good catch. Thanks.